internal/payloads/validation/id.go

```diff
@@ -0,0 +1,26 @@
+package validation
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+)
+
+var (
+	idValidation = regexp.MustCompile("^[0-9]+$")
+	ErrInvalidId = fmt.Errorf("invalid id")
+)
+
+func DigitsOnly(id string) error {
+	if !idValidation.MatchString(id) {
+		return ErrInvalidId
+	}
+
+	// Checking for out of range error
+	_, err := strconv.ParseInt(id, 10, 64)
+	if err != nil {
+		return ErrInvalidId
```

Comment:
```suggestion
		return fmt.Errorf("%w: %s", ErrInvalidId, err.Error())
```

With Go 1.21 you could actually wrap both errors but we are on 1.20 at the moment so string will do.